tests/scripts/final_rc_checks/final_rc_checks.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import json
+import argparse
+
+# Set parser variable to reduce redundancy
+parser = argparse.ArgumentParser()
+
+# -fqdn FQDN -u Username -p Password -api API_Bearer_token
+parser.add_argument("-fqdn", "--rancherURL", dest = "fqdn", help="Rancher FQDN e.g. https://ranchertest.qa.rancher.space")
```

Comment:
nit; can you just have the help say `help="Rancher FQDN, e.g. https://<Rancher FQDN>"`?